modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -233,17 +231,15 @@ const applyFormat = (ed: Editor, name: string, vars?: FormatVars, node?: Node |
           // End the last wrapper
           currentWrapElm = null;
         }
-
       };
 
-      // Process siblings from range
-      each(nodes, process);
+      Arr.each(nodes, process);
```

Comment:
Is there any reason this `.each` was changed to `Arr.each` and not the others?

Reply:
Many others are objects or arrays, as the `Tools.each` variant works with both. Additionally, as I said this doesn't solve every case, only some of them.